packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/network/onboarding/dedicatedCloud-datacenter-network-onboarding.html

```diff
@@ -0,0 +1,42 @@
+<manager-on-boarding-layout
+    data-guides=":: $ctrl.guides"
+    data-image-source=":: $ctrl.illustration"
+>
+    <onboarding-title>
+        <p
+            data-translate="dedicatedCloud_datacenter_network_onboarding_title"
+        ></p>
+    </onboarding-title>
+    <onboarding-description>
+        <div
+            data-translate="dedicatedCloud_datacenter_network_onboarding_description1"
+        ></div>
+        <div
+            data-translate="dedicatedCloud_datacenter_network_onboarding_description2"
+            class="mt-2 mb-4"
+        ></div>
+        <div class="d-flex justify-content-center">
+            <a
+                class="oui-button oui-button_primary oui-button_l text-light"
+                data-translate="dedicatedCloud_datacenter_network_onboarding_cta_start"
+                href="{{:: $ctrl.orderLink }}"
```

Comment:
```suggestion
                data-ng-href="{{:: $ctrl.orderLink }}"
```

This allow disable action if link is wrong
https://docs.angularjs.org/api/ng/directive/ngHref#!%2F